.github/workflows/comment-changed-filenames.yml

```
@@ -0,0 +1,79 @@
+name: Flag changed filenames
+
+# **What it does**: Adds or removes a comment if a PR renames or removes a file.
+# **Why we have it**: Highlights when we need redirects covering certain file paths.
+# **Who does it impact**: PCX team, other Cloudflare contributors
+
+on:
+  pull_request:
+    branches:
+      - production
+
+jobs:
+  
+  flag_changed_filenames:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Get Renamed or Removed Files from Pull Request
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          files=$(curl -s -H "Authorization: token $GITHUB_TOKEN" \
+            -H "Accept: application/vnd.github.v3+json" \
+            "https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/files" | \
+            jq -r '.[] | select(.status=="renamed" or .status=="removed") | select(.filename | endswith(".md")) | if .status == "renamed" then .previous_filename else .filename end' | \
+            sed -e 's|^content||' -e 's|/_index\.md$|/|' -e 's|\.md$|/|')
```

```suggestion
            sed -e 's|^content||' -e 's|/_index\.md$|/|' -e 's|/index\.md$|/|' -e 's|\.md$|/|')
```
Let's handle one more case: `index.md` without an initial underscore.
Unfortunately we will not handle Support paths properly.